executor/aggregate.go

```diff
@@ -795,6 +801,15 @@ func (e *StreamAggExec) Open(ctx context.Context) error {
 	return nil
 }
 
+// Close implements the Executor Close interface.
+func (e *StreamAggExec) Close() error {
+	if err := e.baseExecutor.Close(); err != nil {
```

Comment:
We could firstly release the resources handled by itself, then release the resources handled by its children.